### PR TITLE
Get members and trasform to profiles

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: TrelloMigrate/Program.fs [hunk 1]
```diff
@@ -1,39 +1,18 @@
...
+    let saveData path data = 
+        let result = JsonConvert.SerializeObject(data, Formatting.Indented)
+        File.WriteAllText(path, result)
     try 
         JsonSettings.setDefaults()
-        let trelloCred = Credentials.getTrelloCredentials()
-        let saveFile = @"SrmBoardDownload.json"
-        let srmBoard = downloadSrmBoard trelloCred
-        saveData srmBoard saveFile
+        let srmOutputFile = @"SrmApiModelsPreImport.json"
```
Comment (pallenSL) [inline TrelloMigrate/Program.fs]: This is a path, stored as a string.  You even named it "path" as a parameter to saveData.
Calling it whateverFile here makes me stop and try to work out how/when you opened the file.  Whole seconds of my life I cannot have back!  :D

File: TrelloMigrate/TrelloClient.fs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿module TrelloClient
+
+open Credentials
+open System
+open TrelloModels
+
+//Note: If number of Cards get higher than 1000, then paging needs to be implemented. 
+let getBasicCards trelloCred =
```
Comment (pallenSL) [inline TrelloMigrate/TrelloClient.fs]: private?

File: TrelloMigrate/TrelloClient.fs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+open System
+open TrelloModels
+
+//Note: If number of Cards get higher than 1000, then paging needs to be implemented. 
+let getBasicCards trelloCred = 
+    sprintf "https://api.trello.com/1/boards/524ec750ed130abd230011ab/cards/open?fields=id,name,idMembers,due&key=%s&token=%s" trelloCred.Key trelloCred.Token
+    |> Uri
+    |> JsonHttpClient.get<BasicCard []>
+
+let getBasicMembers trelloCred =
```
Comment (pallenSL) [inline TrelloMigrate/TrelloClient.fs]: private?

File: TrelloMigrate/Transform.fs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        failwith message   
+
+let private getImageUrl avatarHash =   
+    if String.IsNullOrWhiteSpace avatarHash then
+        "https://placebear.com/50/50"
+    else 
+        sprintf "https://trello-avatars.s3.amazonaws.com/%s/50.png" avatarHash
+
+let memberToProfile (bMember : BasicMember) : Profile = 
+    let forename, surname = parseFullName bMember.FullName
```
Comment (pallenSL) [inline TrelloMigrate/Transform.fs]: I think this usage shows that parseFullName isn't right.
Surely we should be assigning 
forename  = getForename bMember.FullName,
surname  = getSurname bMember.FullName,

Comment (jebbinSCL) [inline TrelloMigrate/Transform.fs]: Why does the usage show that it isn't right? 
What's the benefit of having 2 functions? They would be mostly the same except for one having `split.[0]` and the other having `split |> Array.skip 1 |> String.concat " "` after checking the split length.

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**